File: JUNGOL/Beginner/#1291 구구단/구구단.py
s,e = map(int, input().split())
while s<2 or s>9 or e<2 or e>9:
    print("INPUT ERROR!")
    s,e = map(int, input().split())

for i in range(1, 10):
    if s < e:
        for j in range(s, e+1):
            print("%d * %d = %2d"%(j, i, j*i), end="   ")
        print()
    else:
        for j in range(s, e-1, -1):
            print("%d * %d = %2d"%(j, i, j*i), end="   ")
        print()

File: JUNGOL/Beginner/#1291 구구단/구구단.cpp
#include <iostream>
using namespace std;

int main(){
	int s,e; cin>>s>>e;
    while(s<2 || s>9 || e<2 || e>9){
        cout<<"INPUT ERROR!\n";
        cin>>s>>e;
    }

    for(int i=1; i<=9; i++){
        if(s < e){
            for(int j=s; j<=e; j++)
                printf("%d * %d = %2d   ", j, i, j*i);
            cout<<endl;
        }
        else{
            for(int j=s; j>=e; j--)
                printf("%d * %d = %2d   ", j, i, j*i);
            cout<<endl;
        }
    }
}
